release.sh

```diff
@@ -73,7 +73,15 @@ $sudo pip3 install btest wheel
 install_zeek_package brimdata/geoip-conn c9dd7f0f8d40573189b2ed2bae9fad478743cfdf
 install_zeek_package salesforce/hassh 76a47abe9382109ce9ba530e7f1d7014a4a95209
 install_zeek_package salesforce/ja3 421dd4f3616b533e6971bb700289c6bb8355e707
-echo "@load policy/protocols/conn/community-id-logging" | $sudo tee -a /usr/local/zeek/share/zeek/site/local.zeek
+cat << 'EOF' | $sudo tee -a /usr/local/zeek/share/zeek/site/local.zeek
```

Comment:
No need for `cat` here.
```suggestion
$sudo tee -a /usr/local/zeek/share/zeek/site/local.zeek << 'EOF'
```